src/createContext.ts

```diff
@@ -2,7 +2,7 @@
 
 import { GLOBAL, GLOBAL_KEY_TO_ACCESS_NATIVE_STORAGE } from './constants'
 import { AsyncScheduler } from './schedulers'
-import * as list from './stdlib/list'
+import * as list from './stdlib/thunk_list'
```

Comment:
If `useLazyEval == false`, will './stdlib/list.ts' be used?
If not, can we remove './stdlib/list.ts'?

Reply:
I keep list.ts just because of the "list_to_vector" function is called by createContext.ts, which isn't a butin function for js-slang but a internal functionused for create context. And I don't want get confused with this function and the thunk-list implementation

Reply:
ok, got it